### PR TITLE
change version naming

### PR DESCRIPTION
**Pull request by bgruening**
* [ ] I have read the guidelines above.
* [ ] This PR adds a new recipe.
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by bgruening** [comment]
ping @johanneskoester

File: recipes/openbabel/2.3.90dev7d621d9/meta.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 package:
   name: openbabel
-  version: 2.3.2_7d621d9
+  version: 2.3.90dev7d621d9
```
Comment (johanneskoester) [inline recipes/openbabel/2.3.90dev7d621d9/meta.yaml]: Is the 90 coming from the source, or some kind of trick :-)?

Comment (bgruening) [inline recipes/openbabel/2.3.90dev7d621d9/meta.yaml]: its coming from the source. Discovered it today ... :)

Comment (johanneskoester) [inline recipes/openbabel/2.3.90dev7d621d9/meta.yaml]: Great! Let me know once I should delete the old packages (and which ones).

Comment (bgruening) [inline recipes/openbabel/2.3.90dev7d621d9/meta.yaml]: Ok, will do! :)

**Pull request merged**